rootex/framework/components/visual/camera_visual_component.cpp

```diff
@@ -1,24 +1,77 @@
 #include "camera_visual_component.h"
+#include "systems/render_system.h"
+Component* CameraVisualComponent::Create(const JSON::json& componentData)
```

Comment:
Do not load Position and rotation from serialized file, used setup() in TransformComponent, refer conversation on slack